wikibrain/wikimedia_link_issue_reporter.py

```diff
@@ -691,7 +696,7 @@ def check_is_object_brand_is_existing(self, tags):
                 error_id=state,
                 error_general_intructions=error_general_intructions,
                 error_message=message,
-                prerequisite={'wikidata': present_wikidata_id}
+                prerequisite={'wikidata': ';'.join(dissolved_brands)}
```

Comment:
one problem: `prerequisite` is for matching whether such tag is there (maybe I should have documented it?) so QA filter/bot can easily discard cases where data is stale.

Here it should be present_wikidata_id

Maybe info which ones failed can go into `message` field?

Reply:
(I admit that I do not remember exact behaviour of `message` field, would need to look at other code or test it)

Reply:
Ahh that makes sense - I can update that

Reply:
That is now added